.konflux/rpms.in.yaml

```diff
@@ -0,0 +1,19 @@
+contentOrigin:
+  repofiles:
+    - ./ubi.repo
+packages:
+  - cargo
+  - git-core
+  - gcc
+  - jq
+  - python3
+  - python3-pip
+  - python3-setuptools
+  - python3-devel
+  - rubygem-bundler
+  - rubygem-json
+  - subscription-manager
+context:
+  containerfile:
+    file: Dockerfile.konflux
+    stageName: ubi
```

Comment:
nitpick: This line might be obvious to some who have used the prototype generator several times already, but I feel like adding a brief commentary on what this setting does AND/OR why it's necessary. Just my 2c, others might feel differently about it.